.github/dependabot.yml

```diff
@@ -4,7 +4,7 @@
 version: 2
 updates:
   - package-ecosystem: "pip"
```

Comment:
I need to readup on poetry feature but want to comment whether that needs to be specified here instead of pip?

Reply:
This ecosystem is correct for poetry also, according to the docs